tests/data/cdl/data.py

```diff
@@ -37,12 +37,12 @@ def create_file(path: str, dtype: str, num_channels: int) -> None:
 
```

Comment:
One more change left: https://github.com/microsoft/torchgeo/pull/429#discussion_r815087681

This will test that the cmap can be read from the data file itself.